integration_tests/src/main/python/json_test.py

```diff
@@ -359,3 +359,12 @@ def test_json_read_count(spark_tmp_path, v1_enabled_list):
     assert_gpu_and_cpu_row_counts_equal(
             lambda spark : spark.read.schema(schema).json(data_path),
             conf=updated_conf)
+
+def test_from_json_map():
+    # The test here is working around some inconsistencies in how the keys are parsed for maps
+    # on the GPU the keys are dense, but on the CPU they are sparse
+    json_string_gen = StringGen(r'{"a": "[0-9]{0,5}"(, "b": "[A-Z]{0,5}")?}')
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark : unary_op_df(spark, json_string_gen) \
+            .select(from_json(f.col('a'), 'MAP<STRING,STRING>')),
```

Comment:
sorry I missed that you have an extra `f` in `f.col` and changed the import statement I had suggested in my   https://github.com/NVIDIA/spark-rapids/pull/7494#discussion_r1068484468

I believe that `f.col` works in this file only accidentally. because some other pytest files have `import pyspark.sql.functions as f`  and it's picked up when pytest collects tests. I'll post a follow-up PR